test/ui/spec/support/jasmine.json

```diff
@@ -4,7 +4,12 @@
     "**/*.[sS]pec.js"
   ],
   "helpers": [
-    "helpers/**/*.js"
+    "helpers/**/*.js",
+    "node_modules/jquery/dist/jquery.min.js",
```

Comment:
Aha! These are supposed to be relative to the spec directory! https://github.com/jasmine/jasmine/issues/1109#issuecomment-239951622